lib/chips_labels/assistive/clickable/elevated_click/elevated_click_widget.dart

```diff
@@ -0,0 +1,104 @@
+import '/chips_labels/icon_type/type_icon/type_icon_widget.dart';
+import '/flutter_flow/flutter_flow_theme.dart';
+import '/flutter_flow/flutter_flow_util.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'elevated_click_model.dart';
+export 'elevated_click_model.dart';
+
+class ElevatedClickWidget extends StatefulWidget {
+  const ElevatedClickWidget({
+    super.key,
+    this.icon,
+    this.labelText,
+    Color? color,
+    bool? isLabelIcon,
+    required this.action,
+  })  : color = color ?? const Color(0xFFEAEEF4),
+        isLabelIcon = isLabelIcon ?? true;
```

Comment:
This line of code contradicts the optional nature of the parameter, which in turn is causing the app to throw an error at runtime.
```suggestion
      this.isLabelIcon = false,
```